its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -137,7 +137,9 @@ public static ScannerForMSBuild createEndStep(Path projectDir) {
   }
 
   public static ScannerForMSBuild createBeginStep(String projectName, Path projectDir) {
-    return createBeginStep(projectName, projectDir, "");
+    return createBeginStep(projectName, projectDir, "")
+      // FIXME: Remove the next line after this issue is resolved: https://sonarsource.slack.com/archives/CJ8L44WRF/p1704444673166759
```

Comment:
I wonder why this doesn't break the QG...

Reply:
Good point, I left the comment specifically to have the code smell and not forget about it 🤔 
Maybe we are not analyzing the ITs?
I'll take a look a bit later.

Reply:
Yes. And nitpick (not important enough for another CI iteration): we use ToDo for these things.

We use FIXME just during coding for stuff we want to fix before merging a PR.

This is also how the two rules are describing FIXME and ToDo

S1134 - bug you want to fix
[S1135](https://sonarsource.github.io/rspec/#/rspec/S1135) - leftover

Reply:
indeed we only raise on TEST code issues which are only TEST scope
This is both MAIN and TEST so it won't be raised.